### ISSUE TITLE
return call not working

### ISSUE THREAD

**Issue by askmike** [mileszim/webpurify#5]
_(Using node v5.10.1)_

```
var WebPurify = require('./dist/webpurify');

var wp = new WebPurify({
    api_key: 'x'
});

var profane = 'some profane fuck text asshole';

console.log('call return with text:\n', profane)
wp.return(profane)
.then(function(profanity) {
  console.log('result:\n', profanity);
});
```

yields:

```
➜  webpurify git:(master) ✗ node a
call return with text:
 some profane fuck text asshole
result:
 []
```

**Comment by CbCaio** [comment]
Indeed return is not working, I faced the same issue here and also tested your solution, worked great :)
thanks @askmike :+1:

**Comment by mileszim** [comment]
closing, fixed in #8. thanks for reporting!
